internal/cmd/ocean/update_cluster_k8s.go

```diff
@@ -133,6 +133,7 @@ func (x *CmdUpdateClusterKubernetesOptions) initFlags(fs *pflag.FlagSet) {
 		fs.IntVar(&x.DrainingTimeout, flags.FlagOceanDrainingTimeout, x.DrainingTimeout, "")
 		fs.BoolVar(&x.UtilizeReservedInstances, flags.FlagOceanUtilizeReserveInstances, x.UtilizeReservedInstances, "")
 		fs.BoolVar(&x.FallbackToOnDemand, flags.FlagOceanFallbackOnDemand, x.FallbackToOnDemand, "")
+		fs.StringVar(&x.ClusterOrientation, flags.FlagOceanClusterOrientation, x.ClusterOrientation, "")
```

Comment:
ClusterOrientation is an object, not a string- https://docs.spot.io/api/#tag/Ocean-AWS/operation/OceanAWSClusterCreate